shard.yml

```diff
@@ -8,7 +8,6 @@ targets:
   mitm:
     main: src/main.cr
 
-# TODO: Update to "~> 1" once the time comes.
-crystal: ">= 0.36, < 2"
+crystal: 1.3.2
```

Comment:
why not to keep ">= 1.3.2, < 2". I would like to avoid changing it every time we update crystal

Reply:
Just not to forget)

Reply:
and for uniformity)

Reply:
we actually need to update the Crystal version only for engine and tester (high-level app). If there is some issue in other shards we will get this on building the application. 
Moreover, we don't have any specs in mitm which will check that there is no problem with the new Crystal version, so I actually don't see the reason to pin a specific version in our shards.